pkg/version/version.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"encoding/json"
+	"fmt"
+	"runtime"
+	"strings"
+	"text/tabwriter"
+)
+
+var (
+	gitVersion   string // semantic version, derived by build scripts
+	gitCommit    string // sha1 from git, output of $(git rev-parse HEAD)
+	gitTreeState string // state of git tree, either "clean" or "dirty"
+	buildDate    string // build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+)
+
+type Info struct {
+	GitVersion    string `json:"gitVersion,omitempty"`
+	GitCommit     string `json:"gitCommit,omitempty"`
+	GitTreeState  string `json:"gitTreeState,omitempty"`
+	BuildDate     string `json:"buildDate,omitempty"`
+	GoVersion     string `json:"goVersion,omitempty"`
+	AwsSdkVersion string `json:"awsSdkVersion,omitempty"`
```

Comment:
🤷‍♂ 🙃

Reply:
We have to export those variables to be able to serialize them to JSON. :man_shrugging:

Reply:
I am questioning, if we need `AwsSdkVersion`, regardless if it's exported or not :)

Reply:
Got it. I will remove it in an follow-up PR. 🙂